addons/refuel/functions/fnc_refuel.sqf

```diff
@@ -129,7 +129,8 @@ if (_maxFuelTank == 0) then {
         [QGVAR(stopped), [_source, _sink, _nozzle]] call CBA_fnc_localEvent;
         _nozzle setVariable [QGVAR(lastTickMissionTime), nil];
         _nozzle setVariable [QGVAR(isRefueling), false, true];
-    };
+        [_nozzle, QGVAR(nozzle_stop), nil, true, true, true] call CBA_fnc_globalSay3D;
```

Comment:
this is a little ugly (needed in 3 spots), 
but would let us merge this now so we could release CBA and ACE at same time without hard-requiring

```suggestion
        if (isNil "CBA_fnc_getPerFrameHandlerDelay") exitWith {}; // Remove after next release and increase required cba
        [_nozzle, QGVAR(nozzle_stop), nil, true, true, true] call CBA_fnc_globalSay3D;
```

Reply:
I'd add a TODO in that comment, so if we forget to remove it immediately, it will marked, imo, more clearly.